src/App.tsx

```diff
@@ -2,8 +2,13 @@ import { Routes, Navigate, Route } from 'react-router-dom'
 import Register from './views/registration.view'
```

Comment:
I'm seeing marks like **<<<<<<< HEAD** and **>>>>>>> c8df23c8238d86ffaafe8043b12aea02718b3176** this file. 
This usually means there is a git conflict. Please resolve the conflicts